### ISSUE TITLE
player.log(name, jsonData) loses 'name' somewhere before export

### ISSUE THREAD

**Issue by JamesPHoughton** [empiricaly/meteor-empirica-core#78]
The name information associated with a log event does not show up in the exported `player-logs.json` or `player-logs.csv`. 

## Expected Behavior
If I create a log using the implied syntax:
```javascript
    player.log("test", {hi:{nested:true, dateobject:new Date()}, there:'Joe'})
```
and then look at the exported `player-logs.json`, I expect to see a field {..., "name":"test" ...} or similar.
if I look in `player-logs.csv` I expect to see a "name" column.

## Current Behavior
The resulting json entry is:
```json
{"_id":"wwvuL8wxY8Sd5X763",
 "playerId":"DMYtB4iMxYcWn6sGk",
 "gameId":"WtbheahE7pS3No9LX",
 "roundId":"R7kRxeaWvXTMWW6k8",
 "stageId":"QuaqZqcp3wx2TAxpJ",
 "jsonData":"{\"hi\":{\"nested\":true,
                      \"dateobject\":\"2019-12-09T21:36:35.347Z\"
                      },
              \"there\":\"Joe\"}",
 "createdAt":"2019-12-09T21:36:35.361Z"
}
```
and the csv shows:

_id | playerId | gameId | roundId | stageId | jsonData | createdAt
-- | -- | -- | -- | -- | -- | --
wwvuL8wxY8Sd5X763 | DMYtB4iMxYcWn6sGk | WtbheahE7pS3No9LX | R7kRxeaWvXTMWW6k8 | QuaqZqcp3wx2TAxpJ | {"hi":{"nested":true,"dateobject":"2019-12-09T21:36:35.347Z"},"there":"Joe"} | 2019-12-09T21:36:35Z

## Possible Solution
Could either be happening when the log is saved, or when it's exported from the DB.

## Steps to Reproduce (for bugs)
The above example should reproduce

## Context
An easy workaround is to make every log entry a dictionary

## Your Environment

- empirica 1.3.4
- meteor 1.8.2
- macOS

**Comment by JamesPHoughton** [comment]
Looks like it was just left off the export list:

https://github.com/empiricaly/meteor-empirica-core/blob/65f975dbe95ab56d6cc05cecd70bcdb8eeb7db83/startup/server/export.js#L447-L455

See if I can fix this and PR.

**Comment by JamesPHoughton** [comment]
Well, it updates to the db properly, as shown in the mongo shell:
```
meteor:PRIMARY> db.player_logs.find().pretty()
...
{
	"_id" : "xGwxSEMbZCNYXpmKr",
	"playerId" : "TbWmGq7u7Ngf9ZffE",
	"gameId" : "k766PAAoHbaoxmcRZ",
	"roundId" : "FbTvgnegxiYTPk3dQ",
	"stageId" : "J8gpTQWmb4gx8MRWQ",
	"name" : "test",
	"jsonData" : "{\"hi\":{\"nested\":true,\"dateobject\":\"2019-12-09T22:52:34.896Z\"},\"there\":\"Joe\"}",
	"createdAt" : ISODate("2019-12-09T22:52:34.906Z")
}
```
but adding `"name"` to the `playerLogFields` list doesn't seem to be making any difference. (Assuming I'm running my local code properly... I think so.) 

I may be in over my head here - not much experience with databases. @npaton  - what am I missing?

**Comment by amaatouq** [comment]
Make sure you're running your app with the `METEOR_PACKAGE_DIRS` variable set, for example:

```[user@linux]$ METEOR_PACKAGE_DIRS=/home/location-of-custom-core meteor```

^^that "location of custom core" should be the root folder of your clone of the [meteor-empirica-core](https://github.com/empiricaly/meteor-empirica-core) repo.

**Comment by JamesPHoughton** [comment]
Thanks @amaatouq, that was the trick!

**Comment by npaton** [comment]
The fix was deployed in 1.4.0, you can update with `meteor update empirica:core`. Thank you for the fix @JamesPHoughton! ⛵️
